### DISCUSSION TITLE
Stream works on 2.0.0 beta 7 but screen is black

### DISCUSSION THREAD

**Discussion by atheparke** [unknownskl/xbox-xcloud-client#830] [Q&A]
I am on a 2017 Macbook pro. When I sign in, my xbox shows up and I am able to click on start stream. When the stream starts, my screen is black. The menu button opens the menu on my xbox and that about it. any reason why my screen is black and im not able to see the stream?

**Comment by rackerby** [comment]
What options are shown on the Xbox menu?

**Comment by uahero** [comment]
Hi, I had a similar issue with streaming via greenlight on Mac and even with Xbox app on iPad. I fixed it by going to the Video setting in the console and disabling all CEC features. I recon issues also might be with an old or poor HDMI cable from the console to the TV. I hope this helps.
Update: it appears that the issue with the black screen is still on. Proper streaming works only after the console restart :(

**Comment by bmartinis12** [comment]
Hello, I am still currently having this error. Did you end up fixing it?

**Comment by unknownskl** [comment]
Some users reported that this has to do with the resolution, mostly when running on 720p. Is this also the case here?
